### PR TITLE
#540 fix lookup service not implemented GetTopicsOfNamespace

### LINKED ISSUE #540: http lookup service not have namespace topics query implemented
#### Expected behavior

When using regex topics consumer with HTTP client URL, the client will fail to query namespace topics and throw failed handshake error. This is because the client sends protobuf requests to HTTP service. 

#### Actual behavior

get namespace topic lists from http lookup service with http URL.

### PR DESCRIPTION
**Pull request by freeznet**
Fixes #540

### Motivation

When using regex topics consumer with HTTP client URL, the client will fail to query namespace topics and throw failed handshake error. This is because the client sends protobuf requests to HTTP service.

### Modifications

add GetTopicsOfNamespace to LookupService interface
add related tests

### Verifying this change

- [x] Make sure that the change passes the CI checks.

### CODE REVIEW

**Review by congbobo184 — APPROVED** [review]

**Review by codelipenghui — APPROVED** [review]

File: pulsar/internal/namespace_name.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+// under the License.
+
+package internal
+
+import "strings"
+
+func IsV2NameSpaceName(namespace string) bool {
+	parts := strings.Split(namespace, "/")
+	// Legacy namespace name that includes cluster name
+	return len(parts) != 3
```
Comment (cckellogg) [inline pulsar/internal/namespace_name.go:25]: should this be `len(parts) == 2` ? What if `ns` or  `ns/1/2/3` are passed in?
Let's be consistent and use `Namespace`

File: pulsar/consumer_regex.go [hunk 2]
```diff
@@ -343,7 +343,7 @@ func (c *regexConsumer) unsubscribe(topics []string) {
 }
 
 func (c *regexConsumer) topics() ([]string, error) {
-	topics, err := c.client.namespaceTopics(c.namespace)
+	topics, err := c.client.lookupService.GetTopicsOfNamespace(c.namespace, internal.Persistent)
```
Comment (cckellogg) [inline pulsar/consumer_regex.go:346]: For regex are only persistent topics supported?

Comment (freeznet) [reply pulsar/consumer_regex.go:346]: not actually for other client like Java, but this PR does not tend to add `non-persistence` and `all` supporting, will do it in another PR.

**Review by wolfstudy — APPROVED** [review]
Thanks @freeznet , LGTM +1

**Comment by wolfstudy** [comment]
@freeznet Please fix the CI error, thanks.

**Review by cckellogg — APPROVED** [review]

**Pull request merged**